templates/gatsby/files/README.md

```diff
@@ -0,0 +1,27 @@
+# Gatsby for Platform.sh
+
+<p align="center">
+<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/gatsby/.platform.template.yaml&utm_content=gatsby&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
+    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
+</a>
+</p>
+
+This template builds a simple application using Gatsby.
+Gatsby is a free and open source framework based on React that helps developers build blazing fast websites and apps.
```

Comment:
```suggestion

Gatsby is a free and open source framework based on React that helps developers build blazing fast websites and apps.
```

Reply:
This is from their marketing I assume?  It sounds too markety for our descriptions.